includes/controllers/class-message.php

```diff
@@ -220,12 +316,34 @@ public function send_message( $request ) {
 			) )->send();
 		}
 
-		return hp\rest_response(
-			201,
-			[
-				'id' => $message->get_id(),
-			]
-		);
+		// Set response.
+		$response = [
+			'id' => $message->get_id(),
+		];
+
+		if ( $request->get_param( '_render' ) ) {
+
+			// Render block.
+			$output = '<div class="hp-grid__item">';
+
+			$output .= ( new Blocks\Template(
+				[
+					'template' => 'message_view_block',
+
+					'context'  => [
+						'message'     => Models\Message::query()->get_by_id( $message->get_id() ),
```

Comment:
Why querying message again, or the initial object is not complete in some way?

Reply:
Yes, initial object date is NULL